src/Campaigns/resources/admin/components/CampaignDetailsPage/Tabs/Overview.tsx

```diff
@@ -1,14 +1,22 @@
+import {useState} from 'react';
 import CampaignStats from '../Components/CampaignStats';
+import {getCampaignOptionsWindowData, handleTooltipDismiss} from '@givewp/campaigns/utils';
+import CampaignFormNotice from '@givewp/campaigns/admin/components/CampaignDetailsPage/Components/Notices/CampaignForm';
 
 /**
  * @unreleased
  */
 export default function CampaignDetailsOverviewTab() {
+    const campaignWindowData = getCampaignOptionsWindowData();
+    const [showTooltip, setShowTooltip] = useState(campaignWindowData.admin.showCampaignFormNotice);
+    const dismissTooltip = () => handleTooltipDismiss('givewp_campaign_form_notice').then(() => setShowTooltip(false))
```

Comment:
This seems repetitive and includes a lot of imports to keep track of. What if we create a custom hook for this?

Reply:
Make sense

Reply:
Resolved: https://github.com/impress-org/givewp/pull/7818/commits/7cc81df34b79bd4ff4e5d7456f7a2c72f3c70d02